repo/packages/H/history/0/config.json

```diff
@@ -0,0 +1,31 @@
+{
+  "$schema": "http://json-schema.org/schema#",
+  "type": "object",
+  "properties": {
+    "mesos/master/url": {
```

Comment:
The other frameworks have converged on the key `mesos/url`.  It will be easier for clients to marshal config values that can be shared if packages standardize on some common keys.  This needs to be formalized here in the universe spec, but for now it would be nice to follow that convention here.  See https://github.com/mesosphere/universe/blob/master/repo/packages/M/marathon/0/config.json#L8